docs/examples.md

```diff
@@ -45,6 +45,21 @@ data-validation validate column -sc my_bq_conn -tc my_bq_conn -tbls bigquery-pub
 ````shell script
 data-validation validate row -sc my_bq_conn -tc my_bq_conn -tbls bigquery-public-data.new_york_citibike.citibike_stations --primary-keys station_id --hash '*'
 ````
+#### Run a row hash validation with a **NUMERIC** primary key
+For Oracle and Postgres connections, numeric primary keys may not match between source and target data types due to precision or scale discrepancies. You may need to cast the primary key to a string for a valid match which can be done by either of following: 
+* Adding the code snippet above [this](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/63d6303ae08e4f8eb9a663a14d6d60828947f027/data_validation/config_manager.py#L187):
```

Comment:
I think we should remove this option since users would need to clone the repo and locally package this change to use it.